assignments/array-methods.js

```diff
@@ -56,21 +56,40 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(element){
```

Comment:
You can name the parameters anything you want - element definitely works. In the future, however, with more complex problems, using meaningful parameters creates code that is easier to decipher. Maybe instead, you could call these what they are - runner. This way, it reads more like English - for each runner, take the full name array and push in the runner's first name, then the last name.